src/cmd/manage_nsfs.js

```diff
@@ -453,7 +454,7 @@ async function update_account(data, is_flag_iam_operate_on_root_account) {
     const new_name = data.new_name;
     const cur_access_key = has_access_keys(data.access_keys) ? data.access_keys[0].access_key : undefined;
     const update_name = new_name && cur_name && data.new_name !== cur_name;
-    const update_access_key = data.new_access_key && cur_access_key && data.new_access_key !== cur_access_key;
+    const update_access_key = data.new_access_key && cur_access_key;
```

Comment:
I would keep the condition with unwrap() for skipping unneeded unlink + link

Reply:
@shirady why do you need to check that the secret key changed for making a decision about re-linking the file?

Reply:
It is not for the re-linking decision.
I thought of a scenario where an IAM account (username and 1 access key) is created using the IAM API and someone uses the noobaa-cli update and changes only the secret key - the object `access keys` will not be replaced in this condition (unlike the second condition), I will add this handling in the code.